app/services/qa/linked_data/authority_url_service.rb

```diff
@@ -37,29 +42,36 @@ def action_request_variable(action_config, action)
             action_config.qa_replacement_patterns[key]
           end
 
-          def supports_subauthorities?(action_config)
-            action_config.supports_subauthorities?
-          end
-
           def action_subauth_variable(action_config)
             action_config.qa_replacement_patterns[:subauth]
           end
 
-          def action_subauth_variable_value(action_config, subauthority)
-            action_config.subauthorities[subauthority.to_sym]
-          end
-
-          def supports_language_parameter?(action_config)
```

Comment:
Since this is only called once from a private method, it is collapsed into the caller.